nightly-5.8/windows/10.0.20348.1487/Dockerfile

```diff
@@ -0,0 +1,61 @@
+# escape=`
+
+FROM mcr.microsoft.com/windows/servercore:10.0.20348.1487 AS windows
+
+LABEL maintainer="Swift Infrastructure <swift-infrastructure@forums.swift.org>"
+LABEL description="Docker Container for the Swift programming language"
+
+ARG GIT=https://github.com/git-for-windows/git/releases/download/v2.36.0.windows.1/Git-2.36.0-64-bit.exe
+ARG PYTHON=https://www.python.org/ftp/python/3.10.4/python-3.10.4-amd64.exe
+ARG SWIFT_SNAPSHOT=https://download.swift.org/swift-5.8-branch/windows10/swift-5.8-DEVELOPMENT-SNAPSHOT-2023-01-09-a/swift-5.8-DEVELOPMENT-SNAPSHOT-2023-01-09-a-windows10.exe
```

Comment:
We need to change this to using latest Windows toolchain. Let me see if we expose this info easily.